MessageBird/Objects/Request.cs

```diff
@@ -0,0 +1,46 @@
+using System;
+using System.Text;
+
+namespace MessageBird.Objects
+{
+    public class Request
+    {
+        public string Timestamp { get; }
```

Comment:
I would prefer not making these properties part of the public API. Can we make this `internal`?

Reply:
Also, SortedQueryParameters()?

Reply:
Yes please!